### PR TITLE
feat: added the External Link to Documentation button to the Editor

### PR DESCRIPTION
**Pull request by Toxer10**
## Changes

- added the External Link to Documentation button to the Editor

## Screenshots

- image

### CODE REVIEW

**Comment by Toxer10** [comment]
Check please if the approach of storing the links to documentation is okay

File: src/utils/shell.ts [hunk 1]
```diff
@@ -4,8 +4,19 @@ import * as os from 'os';
 // @ts-ignore
 import shellPath from 'shell-path';
 
+import {ResourceKind} from '@models/resourcekindhandler';
+
 let cachedShellPath: string | undefined;
 
+// Documentation links hash-table
+export const resourceKindDocLinks: {[name: string]: string} = {
+  ConfigMap: 'https://kubernetes.io/docs/concepts/configuration/configmap/',
```
Comment (olensmar) [inline src/utils/shell.ts:13]: let's add this link to the ResourceKindHandler interface - i.e. a helpLink property which we can set on all corresponding handlers - and then query the handler for the current resource for this link!?

Comment (Toxer10) [reply src/utils/shell.ts]: Done!

**Review by olensmar — APPROVED** [review]
lgtm - we can add missing links in separate PR

**Pull request merged**